translate/storage/jsonl10n.py

```diff
@@ -409,6 +409,122 @@ def _extract_units(
             yield from parent
 
 
+I18NextV4Suffixes = ["zero", "one", "two", "few", "many", "other"]
+
+
+class I18NextV4Unit(JsonNestedUnit):
+    """A i18next v4 format, JSON with plurals.
+
+    See https://www.i18next.com/
+    """
+
+    @property
+    def target(self):
+        return self._target
+
+    @target.setter
+    def target(self, target):
+        def get_base(item):
+            """Return base name for plurals"""
+            if "_" in item[0]:
+                plural_base, suffix = item[0].rsplit("_", 1)
+
+                if suffix in I18NextV4Suffixes:
+                    return plural_base
+            return item[0]
+
+        def get_plurals(count, base):
+            if count <= 2:
+                return [base + "_one", base + "_other"][:count]
+            return [f"{base}_{I18NextV4Suffixes[i]}" for i in range(count)]
```

Comment:
Shouldn't this be language-dependent and match CLDR? `translate.lang.data.plural_tags` contains the per language mapping, see for example `GoI18NJsonFile` how it can be handled.